### DISCUSSION TITLE
error: linker link.exe not found

### DISCUSSION THREAD

**Discussion by Guspex** [cargo-generate/cargo-generate#583] [Q&A] [answered]
error: linker `link.exe` not found
  |
  = note: program not found

note: the msvc targets depend on the msvc linker but `link.exe` was not found

note: please ensure that VS 2013, VS 2015, VS 2017, VS 2019 or VS 2022 was installed with the Visual C++ option

error: could not compile `proc-macro2` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `cargo-generate v0.13.1`, intermediate artifacts can be found at `C:\Users\andre\AppData\Local\Temp\cargo-installey0vnb`

Caused by:
  build failed

**Comment by sassman** [comment, marked as answer]
> note: please ensure that VS 2013, VS 2015, VS 2017, VS 2019 or VS 2022 was installed with the Visual C++ option

Is the way to go.
